app/linux/BUILD.gn

```diff
@@ -43,6 +43,15 @@ group("symbol_dist_resources") {
 group("dist_resources") {
 }
 
+if (current_cpu == "arm64") {
+  # Copy the binary for stripping Arm64 binaries as `strip` into a directory
+  # that we can put on the PATH.
+  copy("strip_arm64") {
+    sources = [ "/usr/bin/aarch64-linux-gnu-strip" ]
```

Comment:
we don't want to rely on a system exe that may or may not exist

Reply:
Resolved in 2a05892e03395ae76c3d89265dd353bd01aab7bb.